resources/lib/raiplay.py

```diff
@@ -69,7 +69,7 @@ def getVideoMetadata(self, pathId):
         return response["video"]
     
     def getUrl(self, pathId):
-        pathId = pathId.replace(" ", "%20")
+        pathId = urllib2.unquote(pathId)
```

Comment:
Non riesco a capire il significato di questa modifica. Mi puoi fare un esempio? In generale, mi sembra errato sostituire gli escape  %xx con il carattere equivalente. L'escaping viene fatto apposta.

Reply:
ammetto di non ricordare perché, ma funzionava solo così, con gli unescaped url (se non ricordo male lo testai anche su kodi 17 e funzionava, altrimenti non avrei fatto un pull request)